src/main/java/editCard/EditCardController.java

```diff
@@ -1,22 +1,20 @@
-package editCard;
 /*
  * Author: Albert Yan
  * Date: October 31/22
  * The edit card controller for edit card systems, receives the arguments required to perform an edit card action
```

Comment:
think this needs to be a javadoc right above the class name (to be consistent with other files)

Reply:
Agreed. That can be changed after presentations tomorrow

Reply:
Done.